logprep/util/configuration.py

```diff
@@ -195,36 +215,73 @@ def _verify_input(self, logger):
         try:
             _ = Factory.create(self["input"], logger)
         except FactoryError as error:
-            raise InvalidConnectorConfigurationError(str(error)) from error
+            raise InvalidInputConnectorConfigurationError(str(error)) from error
+        except TypeError as error:
+            msg = self._format_type_error(error)
+            raise InvalidInputConnectorConfigurationError(msg) from error
         except KeyError as error:
             raise RequiredConfigurationKeyMissingError("input") from error
 
     def _verify_output(self, logger):
         try:
             _ = Factory.create(self["output"], logger)
         except FactoryError as error:
-            raise InvalidConnectorConfigurationError(str(error)) from error
+            raise InvalidOutputConnectorConfigurationError(str(error)) from error
+        except TypeError as error:
+            msg = self._format_type_error(error)
+            raise InvalidOutputConnectorConfigurationError(msg) from error
         except KeyError as error:
             raise RequiredConfigurationKeyMissingError("output") from error
 
-    def _verify_pipeline(self, logger: Logger):
+    @staticmethod
+    def _format_type_error(error: TypeError) -> str:
+        msg = str(error)
+        if "missing" in str(error):
+            parameters = re.split(r"argument(s)?:", str(error))[-1].strip()
+            if len(re.split("', '| and ", parameters)) > 1:
+                msg = f"Required options are missing: {parameters}."
+            else:
+                msg = f"Required option is missing: {parameters}."
```

Comment:
If I see it correctly then the only difference is the letter `s` here. How about we just simplify this like this:
```suggestion
            msg = f"Required option(s) are missing: {parameters}."
```
That way you also don't have to write an additional test just for the plural version ^^